.cargo-ci/config

```diff
@@ -8,6 +8,6 @@
 rustflags = ["-Ctarget-cpu=native"]
 rustdocflags = ["-Ctarget-cpu=native"]
 
-[target.wasm32-wasi]
+[target.wasm32-wasip1]
```

Comment:
You probably need to have the same target config for both in case you want to test the MSRV still.